ui/tests/e2e/test.js

```diff
@@ -102,6 +110,19 @@ describe("End-to-end", () => {
     await exportToSaturn_selectedCohort();
   });
 
+  test("Field search - chip selected", async () => {
```

Comment:
nit just "Field search"

Typically "- xxx" denotes a variant. In this case, there is no variant, there's just one test.